src/handlers/GraphQLHandler.ts

```diff
@@ -211,4 +221,33 @@ Consider naming this operation or using "graphql.operation" request handler to i
     console.log('Response:', loggedResponse)
     console.groupEnd()
   }
+
+  /**
+   * Checks if the given operation name is matching with handler specifications
+   */
+  private hasMatchingOperationName(operationName: string | undefined) {
```

Comment:
I'd refrain from any issue-unrelated refactoring to keep the pull request's scope minimal.